packages/cli-repl/src/cli-repl.ts

```diff
@@ -353,7 +281,7 @@ class CliRepl {
     console.log(`Using MongoDB:      ${this.internalState.connectionInfo.buildInfo.version}`);
     console.log(`${this.clr('Using Mongosh Beta', ['bold', 'yellow'])}: ${version}`);
     console.log(`${MONGOSH_WIKI}`);
-    if (!this.disableGreetingMessage) console.log(TELEMETRY);
+    if (!this.config.disableGreetingMessage) console.log(TELEMETRY);
```

Comment:
Is `TELEMETRY` a some form of `TELEMETRY_GREETING_MESSAGE`?

Reply:
It’s 

> To help improve our products, anonymous usage data is collected and sent to MongoDB periodically (https://www.mongodb.com/legal/privacy-policy). You can opt-out by running the `disableTelemetry()` command.

so, yes. :+1: